hwi/yanolja_crawling.py

```
@@ -0,0 +1,216 @@
+# # Yanolja 리뷰 크롤링 및 분석
+# 
+# 이번 노트북에서는 Selenium을 사용하여 Yanolja의 호텔 리뷰 페이지에서 데이터를 크롤링하고, 수집한 데이터에 대해 분석을 진행합니다. 이 과정에서는 웹페이지 로드, 데이터 추출, 텍스트 처리 및 분석 결과를 Excel 파일로 저장하는 작업을 포함합니다.
+
+# ### 1단계: Selenium으로 웹페이지 로드
+# 
+# Selenium을 사용하여 Yanolja 리뷰 페이지를 로드하고, 스크롤을 내려서 더 많은 데이터를 가져옵니다.
+
+# In[144]:
+
+
+get_ipython().system('pip install selenium')
```

Ruff (0.13.1)

Undefined name `get_ipython`

(F821)